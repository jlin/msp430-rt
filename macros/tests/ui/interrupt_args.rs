#![no_main]

use msp430_rt_macros::{entry, interrupt};

#[entry]
fn main() -> ! {
    loop {}
}

#[interrupt(arg)]
fn TIM2() -> ! {
    loop {}
}
