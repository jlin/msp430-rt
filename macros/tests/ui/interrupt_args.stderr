error: This attribute accepts no arguments
  --> tests/ui/interrupt_args.rs:10:1
   |
10 | #[interrupt(arg)]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `interrupt` (in Nightly builds, run with -Z macro-backtrace for more info)
